packages/key4hep-stack/package.py

```diff
@@ -9,14 +9,27 @@ class Key4hepStack(BundlePackage):
     
     version('0.1')
     version('nightly')
+
     
     depends_on('podio@master', when="@nightly")
     depends_on('edm4hep@master', when="@nightly")
     depends_on('K4FWCore@master', when="@nightly")
     depends_on('guinea-pig@master', when="@nightly")
-    depends_on('dd4hep@master', when='@nightly')
     
     depends_on("edm4hep@0.1.0", when="@0.1")
     depends_on("K4FWCore@0.1.0", when="@0.1")
     depends_on('guinea-pig@1.2.2rc', when="@0.1")
+
+    depends_on('lcio')
+    depends_on('lcgeo')
+
+    # 
+    depends_on('root cxxstd=17 +root7 +ssl')
+    depends_on('boost +python')
+
+    # there are known issues with compilers from red hat's devtoolset
+    # which are therefore not supported
+    # https://root-forum.cern.ch/t/devtoolset-gcc-toolset-compatibility/38286
+    conflicts("gcc@8.3.1")
```

Comment:
This should be `conflicts("%gcc@8.3.1")` otherwise spack will not recognize it to be a compiler and not emit the conflict. Maybe we could put the comment into the `msg` argument of `conflicts`? This would then be directly displayed and one would not have to dig through `package.py` of the spec to find the comment.

Reply:
Great suggestion! Fixed.